src/main/java/uk/gov/hmcts/darts/transcriptions/mapper/TranscriptionResponseMapper.java

```diff
@@ -40,4 +44,30 @@ TranscriptionUrgencyResponse mapToTranscriptionUrgencyResponse(TranscriptionUrge
         transcriptionUrgencyResponse.setDescription(transcriptionUrgencyEntity.getDescription());
         return transcriptionUrgencyResponse;
     }
+
+    public static TranscriptionResponse mapToTranscriptionResponse(TranscriptionEntity transcriptionEntity) {
+
+        TranscriptionResponse transcriptionResponse = new TranscriptionResponse();
+        try {
+            transcriptionResponse.setCaseId(transcriptionEntity.getCourtCase().getId());
+            transcriptionResponse.setCaseNumber(transcriptionEntity.getCourtCase().getCaseNumber());
+            transcriptionResponse.setCourthouse(transcriptionEntity.getCourtCase().getCourthouse().getCourthouseName());
+            transcriptionResponse.setDefendants(transcriptionEntity.getCourtCase().getDefendantStringList());
+            transcriptionResponse.setJudges(transcriptionEntity.getCourtCase().getJudgeStringList());
+            if (transcriptionEntity.getTranscriptionDocument() != null) {
+                transcriptionResponse.setTranscriptFileName(transcriptionEntity.getTranscriptionDocument().getFileName());
+            }
+            transcriptionResponse.setHearingDate(transcriptionEntity.getHearing().getHearingDate());
+            if (transcriptionEntity.getTranscriptionUrgency() != null) {
+                transcriptionResponse.setUrgency(transcriptionEntity.getTranscriptionUrgency().getDescription());
+            }
+            transcriptionResponse.setRequestType(transcriptionEntity.getTranscriptionType().getDescription());
+            transcriptionResponse.setTranscriptionStartTs(String.valueOf(transcriptionEntity.getStartTime()));
```

Comment:
See earlier comment in `transcriptions.yaml` - The Jackson serialisation will be handled for you in the response to the ISO DATE TIME string format. So no need to do `String.valueOf` here, once the model is corrected.

Reply:
Done as part of changing timestamp fields to date-time format